ovs-p4rt/ovs_p4rt.cc

```diff
@@ -1465,10 +1465,15 @@ void ConfigFdbTableEntry(struct mac_learning_info learn_info,
       learn_info.is_tunnel = true;
     }
 
-    status_or_read_response =
-        GetL2ToTunnelV6TableEntry(session.get(), learn_info, p4info);
-    if (status_or_read_response.ok()) {
-      learn_info.is_tunnel = true;
+    /* If learn_info.is_tunnel is true, then we dont need to check for v6 table
```

Comment:
Punctuation:

1. **don't** requires an apostrophe. It is a contraction of **do not**.
2. The period following "entry" should be a comma, not a period. It changes the meaning of "as" to "because".

(assuming that the remainder comment is still applicable once you've corrected the sense to **not true**.

Reply:
Sure Derek, modified as suggested.